sqlmesh/core/engine_adapter/base.py

```diff
@@ -199,9 +204,18 @@ def catalog_support(self) -> CatalogSupport:
         return CatalogSupport.UNSUPPORTED
 
     @classmethod
-    def _casted_columns(cls, columns_to_types: t.Dict[str, exp.DataType]) -> t.List[exp.Alias]:
+    def _casted_columns(
+        cls,
+        columns_to_types: t.Dict[str, exp.DataType],
```

Comment:
do we want to be more specific with this name. i.e. `target_columns_to_types`.

Reply:
Currently the engine adapter interface as an implicit understanding that `columns_to_types` is the `target_columns_to_types`. I would hoping the existence of source columns makes that clear without having to rename everything and extending the length of that variable.

Reply:
FWIW, if there's any such understanding at all today, it's very implicit even to me. If you asked me about what `columns_to_types` is before this PR I'd tell you it's the model's `columns_to_types`.

Reply:
You would prefer then to have it renamed within the engine adapter class?

Reply:
sure